### ISSUE TITLE
Create Speech Recognition Plugin for Tizen Web and Wearable

### ISSUE THREAD

**Issue by abcplex** [crosswalk-project/crosswalk-tizen#92]
I have a Samsung Gear 2 Wearable device. I am creating a web app for my device. But I found that the W3c Speech api is not working in Tizen wearable device(although works on simulator). From here https://www.tizen.org/events/tizen-developer-conference/2014/beyond-tizen-apis-using-crosswalk-extension-framework-create-your-own-apis, I found that we can extend the web app using Cross-Walk. So, my question is that how can I getting started. Do you have any samples for achieving this?

**Comment by WonyoungChoi** [comment]
The user defined extension feature will be added into Tizen 3.0 soon. Application developers can include custom extensions into their webapp packages. Plus, speech recognition feature will be supported in Tizen 3.0 also. But unfortunately, Gear2's tizen version is 2.3.1  now. That means that creating those extension is not possible now. :(

**Comment by abcplex** [comment]
@WonyoungChoi thanks

**Comment by abcplex** [comment]
@WonyoungChoi any release date?

**Comment by WonyoungChoi** [comment]
The user define extension feature will be landed in this project next month. But the release date of the Tizen 3.0 is undecided still.

**Comment by abcplex** [comment]
@WonyoungChoi one question I tried this on native(C Language),

```
    bool stt_supported = false;
    system_info_get_platform_bool("http://tizen.org/feature/speech.recognition", &stt_supported);
```

I have also added speech.recognition in manifest file. It says not supported. It means we can't use native and web app for Samsung Gear 2 Tizen 2.3.1 to build speech recognition? Correct?  Any alternative wear you suggest which support speech/voice? Should I try Apple or Android wear?
